Makefile

```diff
@@ -87,6 +87,7 @@ oc-create-cloudigrade:
 	| oc create -f -
 
 oc-forward-ports:
+	-make oc-stop-forwarding-ports 2>/dev/null
```

Comment:
is that `-make` intentional?

Reply:
Yes, the `-` ignores failure for that command, since you won't always have existing forwarded ports.

Reply:
I don't understand but I see it makes it work

Reply:
Normally when any command in a target fails, make will stop and spit out an error. In this case we're bypassing it by telling it that a failure at that point is ok.